tests/cleo_tests/compile.bat

```diff
@@ -0,0 +1,12 @@
+@REM Compile all .txt files in the current directory to .s files using Sanny Builder 4
+@REM Usage: set SANNY="path\to\sanny.exe" && compile.bat
+
+@echo off
+
```

Comment:
Please add
```
for /R %%f in (????.s) do (del /q "%%f")
```

Reply:
you mean, delete existing .s file before compiling new?